tests/unit/sstring_test.cc

```diff
@@ -306,3 +308,13 @@ BOOST_AUTO_TEST_CASE(test_compares_left_hand_not_string) {
     BOOST_REQUIRE(std::string("a") < sstring("b"));
 #endif
 }
+
+#if FMT_VERSION >= 90000
+
+BOOST_AUTO_TEST_CASE(test_range_fmt) {
+    std::ignore = fmt::format("{}", std::optional(sstring{"hello"}));
```

Comment:
might want to guard this very test with `#if FMT_VERSION >= 100000`, and keep the test for `fmt/range` in `#if FMT_VERSION >= 90000`. as the formatter for `std::optional<>` as introduced by fmtlib v10.0.0, see https://github.com/fmtlib/fmt/releases/tag/10.0.0.

this is why the CI testing is failing for this PR -- we only have access to fmt-devel 9.1.0 in the frozen toolchain docker container.

Reply:
ack